### ISSUE TITLE
Filter:  Make "Get Access" Filter options act like a checkbox so that the selected/unselected states are announced

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#628]
## Description

When using a screen reader, selecting a filter under Get Access does not give feedback whether the option has been selected or not. The selected state of these filter options should be announced.

## Recommendations

- use `<role=checkbox">` `<aria-checked="true">` to mimic a multi-select checkbox, rather than making these toggle buttons.
- associate these option to the filter heading so that users knows what they are selecting for.

**Comment by Amberroseweeks** [comment]
Hello, can I please be assigned this issue if no one else is working on this?

Thank you!
